package.json

```diff
@@ -171,13 +172,15 @@
     "@types/marked": "^0.7.4",
     "@types/mock-fs": "^4.10.0",
     "@types/node": "^12.12.45",
+    "@types/node-notifier": "^8.0.0",
     "@types/proper-lockfile": "^4.1.1",
     "@types/react-beautiful-dnd": "^13.0.0",
     "@types/tar": "^4.0.3",
     "array-move": "^3.0.0",
     "chalk": "^4.1.0",
     "conf": "^7.0.1",
     "crypto-js": "^4.0.0",
+    "dateformat": "^4.3.1",
```

Comment:
We have `moment` already for working with `Date` or what's the purpose to add this?